gateway/src/apicast/policy/caching/schema.json

```diff
@@ -1,9 +1,28 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Caching policy configuration",
+  "description":
```

Comment:
@mikz  This is the best way I found to write long descriptions. We'll need to specify in the meta-schema that the descriptions might be strings or arrays of strings.

Reply:
Yep, that is done in https://github.com/3scale/apicast/pull/564, but the description is not part of the schema there.